Project/DAQ_System/Program/Src/Sensor/ECU/PE3/Frames/frame_pe1.hpp

```diff
@@ -0,0 +1,37 @@
+/*
+* ECU PE3 Frame: PE1
+* Author:   William Lim
+*           Cristian Cruz
+*
+* Email:    wll180000@utdallas.edu
+*           cris14.alex@gmail.com
+*
+* (c) 2024 Dallas Formula Racing - Embedded Firmware Team
+* Formula SAE International Collegiate Chapter
+* GPL-3.0 License
+*/
+
+#ifndef ECU_PE3_FRAMES_FRAMEPE1_H
+#define ECU_PE3_FRAMES_FRAMEPE1_H
```

Comment:
This is a rather minor naming thing but I kinda think ECU_PE3_FRAMES_FRAMEPE1_H is redundant. What do you think about ECU_PE3_FRAMES_PE1_H? you already establish that it's a frame so I think FRAMEPE1 is redundant.

Reply:
Sure. I only did it based on the folder locations and file name. But I am open to that.